Library/Currency.swift

```diff
@@ -0,0 +1,56 @@
+import UIKit
+import KsApi
+
+public enum Currency: String, Encodable, CaseIterable {
+  case EUR
+  case AUD
+  case CAD
+  case CHF
+  case DKK
+  case GBP
+  case HKD
+  case JPY
+  case MXN
+  case NOK
+  case NZD
+  case SEK
+  case SGD
+  case USD
+
+  public static var rowHeight: CGFloat {
```

Comment:
Hm, this seems like more of a `View`-related property, can we move it to where it's used instead? Just to keep this `Currency` enum as more of a clean model object.